package.json

```diff
@@ -11,18 +11,20 @@
     "lint:css": "stylelint src/**/*.{css,scss,vue}",
     "lint:commit": "commitlint --from develop --verbose",
     "lint": "cross-env NODE_ENV=production npm run lint:commit && npm run lint:vue && npm run lint:css",
-    "build": "vue-cli-service build",
-    "build:cordova": "cross-env PLATFORM=cordova vue-cli-service build",
-    "build:extension": "cross-env PLATFORM=extension vue-cli-service build",
-    "build:web": "cross-env PLATFORM=web vue-cli-service build",
+    "build": "ionic build",
+    "build:android": "cross-env PLATFORM=ionic ionic capacitor build android",
+    "build:ios": "cross-env PLATFORM=ionic ionic capacitor build ios",
```

Comment:
is there a watch feature by using IONIC for android and IOS debugging?
I remember @smaroudas talking about this